### DISCUSSION TITLE
Option to make links not clickable in the editor

### DISCUSSION THREAD

**Discussion by lolbat** [notaapp/talk#361] [Feature Requests] [answered]
One of the things I quite like about Markdown is that it is easy to to edit and style text.

The current version of Nota makes the text between braces an active link as well as the URL. Which makes it a total PITA to edit either text as most of the time if you click it the link will open in your browser. 

```markdown
There are a few ways to do this but Hugo has a very helpful 
set of [Build options](https://gohugo.io/content-management/build-options/) 
that you can use to help make this type of page easier to build. 
```

So if I want to edit "Build options" then I have to be careful how I click the text or click somewhere else and then use the keyboard to move to where I want to edit

It would be great if there was an option to disable that

Thanks

**Comment by astoilkov** [comment, marked as answer]
We are already working on this and will make a release soon. Take a look at the already existing discussion — https://github.com/notaapp/talk/issues/357.

Sorry for the delayed response.
